src/ansible_navigator/runner/command_async.py

```diff
@@ -1,10 +1,8 @@
-"""Implementation of the asynchronous invocation of ``ansible-runner``.
-
-Herewith lies the ability to invoke ``ansible-runner`` in an async fashion.
-A queue is provided and ``ansible-runner`` uses ``pexpect`` to parse
-standard out and error from the command run and populates the
-queue with messages.
-"""
+"""Implementation of the asynchronous invocation of ``ansible-runner``. \
```

Comment:
I think if all these \ go away it'll complain about one line....

Reply:
I worked that out. The closing quotes just needed to be moved to a separate line below. I was struggling with fixing that in the beginning for the descriptions but with a little trial and error it should be looking much better now and without any /.